packages/i18n-calypso/test/cli/examples/i18n-test-examples.jsx

```diff
@@ -0,0 +1,103 @@
+// // This file is just an example of different translation requests that will be parsed as part of the test. This file itself doesn't need to actually work. :-)
+/*eslint-disable */
+/* global i18n:false, howManyHats, getCity, getZip, React */
+function test() {
+	// simplest case... just a translation, no special options
+	var content = i18n.translate( 'My hat has three corners.' );
+
+	// or it could also be called like this
+	content = i18n.translate( { original: 'My hat has three corners too.' } );
+
+	// pluralization
+	var numHats = howManyHats(); // returns integer
+	content = i18n.translate( {
+		original: {
+			single: 'My hat has four corners.',
+			plural: 'My hats have five corners.',
+			count: numHats
+		}
+	} );
+
+	// Should be merged in POT with the pluralized form
+	content = i18n.translate( { original: 'My hat has four corners.' } );
+
+	// Should catch template literals
+	content = i18n.translate( `My hat has six corners.` );
+	content = i18n.translate( `My hat
+has seventeen
+corners.` );
+
+	// providing context
+	content = i18n.translate( {
+		original: 'post',
```

Comment:
:information_source: This string already exists with the following **contexts**:
- `null` (no context)
- `add new from admin bar`
- `noun`
- `post a comment`
- `post type singular name`
- `Verb, to post`

 Would it make sense to reuse one of the above?